Dockerfile

```diff
@@ -2,33 +2,33 @@ FROM debian:jessie
 MAINTAINER Jan Broer <janeczku@yahoo.de>
 ENV DEBIAN_FRONTEND noninteractive
 
-# Download & install required applications: curl, sudo.
+# Following 'How do I add or remove Dropbox from my Linux repository?' - https://www.dropbox.com/en/help/246
+RUN echo 'deb http://linux.dropbox.com/debian jessie main' > /etc/apt/sources.list.d/dropbox.list
+RUN apt-key adv --keyserver pgp.mit.edu --recv-keys 1C61A2656FB57B7E4DE0F4C1FC918B335044912E
 RUN apt-get -qqy update
-RUN apt-get -qqy install wget python sudo
-
-# Create service account and set permissions.
-RUN groupadd dropbox
-RUN useradd -d /dbox -c "Dropbox Daemon Account" -s /usr/sbin/nologin -g dropbox dropbox
-RUN mkdir -p /dbox/.dropbox /dbox/.dropbox-dist /dbox/Dropbox /dbox/base
-
-# Download & install Dropbox 3.2.9 and latest python client
-RUN wget -nv -O /dbox/base/dropbox.tar.gz "https://github.com/radio-astro/docker-dropbox/blob/master/dropbox-lnx.x86_64-3.2.9.tar.gz?raw=true"
-RUN wget -nv -O /dbox/dropbox.py "https://www.dropbox.com/download?dl=packages/dropbox.py"
```

Comment:
Any particular reason why you did remove the `dropbox.py` command line client? Not useful?